scripts/reset-test-db.sh

```diff
@@ -51,6 +54,14 @@ declare -a database_order
 databases=(["default"]="edxtest" ["student_module_history"]="student_module_history_test")
 database_order=("default" "student_module_history")
 
+calculate_migrations() {
+    echo "Calculating migrations for fingerprinting."
+    output_file="common/test/db_cache/bok_choy_${db}_migrations.yaml"
+    # Redirect stdout to /dev/null because the script will print
+    # out all migrations to both stdout and the output file.
+    ./manage.py lms --settings $SETTINGS show_unapplied_migrations --database $db --output_file $output_file
```

Comment:
Do we still need to redirect the output to /dev/null so that it doesn't clutter the console output?
If so, add that to this line.
If not, then remove the comments above this line.

Reply:
ah, my bad. I had the management command writing to stdout for debugging purposes. I can redirect it back to dev/null